code/lib/blocks/src/blocks/mdx.tsx

```diff
@@ -185,6 +192,9 @@ const HeaderWithOcticonAnchor: FC<PropsWithChildren<HeaderWithOcticonAnchorProps
   const OcticonHeader = OcticonHeaders[as];
   const hash = `#${id}`;
 
+  const copyUrl = new URL(window.parent.location.href);
```

Comment:
Is there a reason why you don't use the whole URL but just the href? E.g. you are not preserving search params, but they could be used to filter content in some docs pages and could be relevant for the person copying the URL to provide an exact replica to the person receiving it.